packages/application-shell-connectors/src/components/project-extension-image-regex/project-extension-image-regex.spec.tsx

```diff
@@ -12,7 +12,7 @@ describe('rendering', () => {
           <ProjectExtensionProviderForImageRegex>
             <div />
           </ProjectExtensionProviderForImageRegex>
-        ).renderProp('children', { loading: true, data: undefined });
+        ).renderProp('children')({ loading: true, data: undefined });
```

Comment:
You'll see a lot of these changes. It was necessary because TS was pointing to the `renderProp` method from `enzyme` (not our own package), which has a slightly different API.
As a consequence, I refactored those and removed our extension package.

Reply:
Will this be a breaking change?

Reply:
I’m afraid so. I tried to find a backwards compatible solution but if we can’t overload the `renderProp` signature of ShallowWrapper there is no way to make it work.

One thing that I haven’t tried is to keep the old version dependency of jest package and use yarn resolutions to point to the new version only for the appkit repo

Reply:
Seems to work with yarn resolutions: https://github.com/commercetools/merchant-center-application-kit/pull/737/commits/c674021cf0bbc17e32b28809d8118776d6165290

Reply:
Nice. Eventually we will have to break this though. It will mostly hit the MC I think. Most custom apps are actually little to  not much tested to my knowledge.

Reply:
Do we still need to make these changes with the yarn resolution?

Reply:
In the appkit repo we can use the new version, but the jest package contains the "old" version so that consumers should not need to change anything (yet).